students/hyska_monika/lesson_02_flow_control/KnightMove.py

```diff
@@ -0,0 +1,18 @@
+# The program print that Knight can change position
+# from one field to second field in one move
+import math
+
+v1 = int(input("Put vertical position for 1st field (from 1 to 8): "))
+h1 = int(input("Put horizontal position for 1st field(from 1 to 8): "))
+v2 = int(input("Put vertical position for 2nd field(from 1 to 8): "))
+h2 = int(input("Put horizontal position for 2nd field(from 1 to 8): "))
+
+vector_1 = int(math.fabs(v1 - v2))
+vector_2 = int(math.fabs(h1 - h2))
+
+if ((vector_1 == 2) & (vector_2 == 1)) or ((vector_1 == 1) & (vector_2 == 2)):
```

Comment:
Coincidentally, this will work, but you are using a bitwise AND operator (`2 & 1 == 3` instead of `2 and 1 == True`), not a Booolean one. This can also affect the order of evaluation. If you use Boolean operators you can drop all the parentheses for sure, the precedence is the one commonly used (`and` has a higher precedence than `or`). Also again the `math.fabs` instead of `abs` and `vector` is a confusing name for the integer. Use `column_diff`, `row_diff`  or something along those lines.